packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -283,11 +286,8 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                   aria-describedby={items.length > 0 ? this._ariaMap.selectedItems : undefined}
                   aria-controls={`${suggestionsAvailable} ${selectedSuggestionAlertId}` || undefined}
                   aria-activedescendant={this.getActiveDescendant()}
-                  autoCapitalize="off"
```

Comment:
Given that autofill has this already, we can probably remove it from both here and extended picker. Same with autocomplete.